### PR TITLE
Community page updates

### PR DESCRIPTION
**Pull request by anderseknert**
* Add OPA LinkedIn
* Add local Meetup links
* Charlie Egan -> Peter Macdonald
* Add Mastodon links

Signed-off-by: Anders Eknert <anders@eknert.com>

### CODE REVIEW

**Comment by anderseknert** [comment]
@Parsifal-M I guess you might want to review this too 🙂 https://deploy-preview-5517--openpolicyagent.netlify.app/community

**Comment by Parsifal-M** [comment]
> @Parsifal-M I guess you might want to review this too slightly_smiling_face https://deploy-preview-5517--openpolicyagent.netlify.app/community

LGTM!! :star_struck:

File: docs/website/static/community.html [hunk 1]
```diff
@@ -227,6 +248,11 @@ <h2 class="divider line glow">Champions</h2>
                     <div class="card-text">
                       <div class="header">John Reese</div>
                       <div class="subheader">
+                        <a href="https://hachyderm.io/@jpreese" alt="Mastodon">
```
Comment (srenatus) [inline docs/website/static/community.html:251]: Do you think we should check with John if he's OK with this getting included? (Or did you already?)

File: docs/website/static/community.html [hunk 2]
```diff
@@ -337,6 +347,72 @@ <h2 class="divider line glow">Contribute</h2>
...
+            </div>
+
+            <div class="card">
+              <div class="card-body">
+                <div class="support-card-content d-flex flex-column align-items-start">
+                  <img title="Meetup" class="card-img align-middle" src="img/community-logos/meetup.png"/>
+                  <div class="card-text">
+                    <div class="header">Host your own!</div>
+                    <div class="subheader">
+                      Host a localized meetup in your area or virtually. Share what you’ve learned and connect with the OPA community!
```
Comment (srenatus) [inline docs/website/static/community.html:405]: [nit] That's a funky single-quote sign there. I guess it's not wrong, just have a tendency to use the simple one, `'`

**Review by srenatus — APPROVED** [review]

File: docs/website/static/community.html (hunk 1 above)
Comment (anderseknert) [reply docs/website/static/community.html:251]: It's a public profile just like his Twitter, but you're right, let's summon @jpreese to check.

File: docs/website/static/community.html (hunk 2 above)
Comment (anderseknert) [reply docs/website/static/community.html:405]: That section was merely moved in this PR, but yeah, I'll have that fixed while I'm at it 👍

Comment (srenatus) [reply docs/website/static/community.html:405]: Thanks!

Comment (anderseknert) [reply docs/website/static/community.html]: Fixed 👍

**Review by charlieegan3 — APPROVED** [review]
This looks fine to me, pending comments from others.

I have some updates for our Hugo flow that I'll share in other PR.

File: docs/website/static/community.html (hunk 1 above)
Comment (jpreese) [reply docs/website/static/community.html:251]: LGTM 🎉

**Review by jpreese — APPROVED** [review]

**Pull request merged**